rootfs/etc/profile.d/aws-vault.sh

```diff
@@ -14,7 +14,7 @@ function _validate_aws_vault_server() {
 
 	if [[ $instance == "aws-vault" ]]; then
 		# We successfully connected to an active aws-vault credentials server, so use it
-		_assume_active_aws_role
```

Comment:
Why the change? Guessing this was done previously to be similar to https://github.com/cloudposse/geodesic/pull/478/files#diff-1ae3ba81cafb63b08cd75d2558472b27R230

Reply:
The leading underscore was intended to hide the function from the namespace of commands people would run on the command line. Turns out this command has utility on the command line (see usage notes in the PR), so I changed the name to reflect that.